graph/pkg/service/v0/drives.go

```diff
@@ -249,22 +249,14 @@ func (g Graph) CreateDrive(w http.ResponseWriter, r *http.Request) {
 }
 
 func (g Graph) UpdateDrive(w http.ResponseWriter, r *http.Request) {
-	// wildcards however addressed here is not yet supported. We want to address drives by their unique
-	// identifiers. Any open queries need to be implemented. Same applies for sub-entities.
-	// For further reading: http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#sec_AddressingaSubsetofaCollection
-
-	// strip "/graph/v1.0/" out and parse the rest. This is how godata input is expected.
-	//https://github.com/CiscoM31/godata/blob/d70e191d2908191623be84401fecc40d6af4afde/url_parser_test.go#L10
-	sanitized := strings.TrimPrefix(r.URL.Path, "/graph/v1.0/")
-
-	req, err := godata.ParseRequest(r.Context(), sanitized, r.URL.Query())
+	driveID := chi.URLParam(r, "driveID")
+	driveID, err := url.PathUnescape(driveID)
```

Comment:
`driveID` is defined and then directly reassigned. This can be a bit confusing. Would you mind doing
```go
driveID, err := url.PathUnescape(chi.URLParam(r, "driveID"))
```
?

Reply:
done